recirq/otoc/loschmidt/tilted_square_lattice/gen-small-v1.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+
+import cirq
+from recirq.otoc.loschmidt.tilted_square_lattice import \
+    get_all_tilted_square_lattice_executables
+
+FN = 'loschmidt.tilted_square_lattice.small-v1.json.gz'
+
+
+def main():
```

Comment:
Should add module level docstring to explain what this is and what the valid args and/or example command are.

Reply:
added (using verbiage from the new description field in the algorithmic_benchmarking_library card catalog)